multiqc/modules/hicexplorer/hicexplorer.py

```diff
@@ -0,0 +1,184 @@
+from multiqc.modules.base_module import BaseMultiqcModule
+import logging
+from collections import OrderedDict
+
+from multiqc import config
+from multiqc.plots import bargraph
+
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule):
+    def __init__(self):
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='HiCExplorer', anchor='hicexplorer',
+                                            href="https://hicexplorer.readthedocs.io",
+                                            info=" addresses the common tasks of Hi-C analysis from processing to visualization.")
+
+        self.mod_data = dict()
+        for file in self.find_log_files('hicexplorer'):
+            self.mod_data[file['s_name']] = self.parse_logs(file['f'])
+            self.add_data_source(file)
+        if len(self.mod_data) == 0:
+            raise UserWarning
+        self.colors = ["#1f77b4",
```

Comment:
Is there a specific reason to use custom colours for this module?

Reply:
I tried different colors for the plot and the current scheme was the one I liked most.